arangod/Aql/ExecutionNode/EnumerateNearVectorNode.cpp

```diff
@@ -187,6 +194,10 @@ EnumerateNearVectorNode::EnumerateNearVectorNode(
       _offset(base.get(kOffset).getNumericValue<std::size_t>()) {
   std::string iid = base.get("index").get("id").copyString();
 
+  if (auto ns = base.get(knProbe); ns.isNumber()) {
+    _searchParameters.nProbe = ns.getNumericValue<std::int64_t>();
+  }
+
```

Comment:
and here we could simply use the deserializer 🙂

Reply:
Ah makes sense!